tinygrad/shape/shapetracker.py

```diff
@@ -109,9 +109,9 @@ def _reshape(view: View, new_shape:Tuple[int, ...]) -> Tuple[View, bool]:
         if x == 1 and y != (0, 1):
           new_mask_tuple = ((0,0),) * len(new_shape)
           break
-      else:
-        new_mask: List[Tuple[int, int]] = [y for x,y in zip(shape, mask) if x != 1]
-        new_mask_tuple = tuple([(0,1) if x == 1 else new_mask.pop(0) for x in new_shape])
+        else:
+          new_mask: List[Tuple[int, int]] = [y for x,y in zip(shape, mask) if x != 1]
+          new_mask_tuple = tuple([(0,1) if x == 1 else new_mask.pop(0) for x in new_shape])
```

Comment:
It's not an indentation issue. It's an else statement for a for loop, it executes when for loop ended without breaking. This change just slows down code since `new_mask` and `new_mask_tuple` are computed on every single iteration, instead of just once.

Reply:
https://book.pythontips.com/en/latest/for_-_else.html